rpcs3/Emu/Cell/PPUModule.cpp

```diff
@@ -1,4 +1,4 @@
-#include "stdafx.h"
```

Comment:
could this be a change in file encoding?

Reply:
It is, but if its now with BOM it should be ok